exercises/protein-translation/example.cpp

```diff
@@ -0,0 +1,29 @@
+#include "protein_translation.h"
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <map>
+
+using namespace std;
+
+vector<string> protein_translation::proteins(string rna) {
+
+    map<string, string> dic;
+    dic["AUG"] = "Methionine";
+    dic["UUU"] = dic["UUC"] = "Phenylalanine";
+    dic["UUA"] = dic["UUG"] = "Leucine";
+    dic["UCU"] = dic["UCC"] = dic["UCA"] = dic["UCG"] = "Serine";
+    dic["UAU"] = dic["UAC"] = "Tyrosine";
+    dic["UGU"] = dic["UGC"] = "Cysteine";
+    dic["UGG"] = "Tryptophan";
+    dic["UAA"] = dic["UAG"] = dic["UGA"] = "STOP";
+
+    vector<string> polypeptide;
+    for (int i = 0; i < (int)rna.size()-2; i += 3) {
```

Comment:
Nit: Should probably use `size_t` instead of `int` here for `i`'s type.

Reply:
I didn't make this change because I'm afraid we can get some king of overflow error

Reply:
Oops, yes you're correct here :) There'd be a pretty bad underflow error if the array was small.